code/modules/mining/equipment/crusher.dm

```diff
@@ -0,0 +1,474 @@
+	if(istype(A, /obj/item/twohanded/required/kinetic_crusher))
+		add_to(A, user)
+	else
+		..()
+
+/obj/item/crusher_trophy/proc/add_to(obj/item/twohanded/required/kinetic_crusher/H, mob/living/user)
+	for(var/t in H.trophies)
+		var/obj/item/crusher_trophy/T = t
+		if(istype(T, denied_type) || istype(src, T.denied_type))
+			to_chat(user, "<span class='warning'>You can't seem to attach [src] to [H]. Maybe remove a few trophies?</span>")
+			return FALSE
+	H.trophies += src
+	to_chat(user, "<span class='notice'>You attach [src] to [H].</span>")
+	playsound(loc, usesound, 100, 1)
+	user.unEquip(src)
+	forceMove(H)
+	return TRUE
+
+/obj/item/crusher_trophy/proc/remove_from(obj/item/twohanded/required/kinetic_crusher/H, mob/living/user)
+	forceMove(get_turf(H))
+	H.trophies -= src
+	return TRUE
+
+/obj/item/crusher_trophy/proc/on_melee_hit(mob/living/target, mob/living/user) //the target and the user
+/obj/item/crusher_trophy/proc/on_projectile_fire(obj/item/projectile/destabilizer/marker, mob/living/user) //the projectile fired and the user
+/obj/item/crusher_trophy/proc/on_mark_application(mob/living/target, datum/status_effect/crusher_mark/mark, had_mark) //the target, the mark applied, and if the target had a mark before
+/obj/item/crusher_trophy/proc/on_mark_detonation(mob/living/target, mob/living/user) //the target and the user
+
+//goliath
+/obj/item/crusher_trophy/goliath_tentacle
+	name = "goliath tentacle"
+	desc = "A sliced-off goliath tentacle. Suitable as a trophy for a kinetic crusher."
+	icon_state = "goliath_tentacle"
+	denied_type = /obj/item/crusher_trophy/goliath_tentacle
+	bonus_value = 2
+	var/missing_health_ratio = 0.1
+	var/missing_health_desc = 10
+
+/obj/item/crusher_trophy/goliath_tentacle/effect_desc()
+	return "mark detonation to do <b>[bonus_value]</b> more damage for every <b>[missing_health_desc]</b> health you are missing"
+
+/obj/item/crusher_trophy/goliath_tentacle/on_mark_detonation(mob/living/target, mob/living/user)
+	var/missing_health = user.maxHealth - user.health
+	missing_health *= missing_health_ratio //bonus is active at all times, even if you're above 90 health
+	missing_health *= bonus_value //multiply the remaining amount by bonus_value
+	if(missing_health > 0)
+		target.adjustBruteLoss(missing_health) //and do that much damage
+
+//watcher
+/obj/item/crusher_trophy/watcher_wing
+	name = "watcher wing"
+	desc = "A wing ripped from a watcher. Suitable as a trophy for a kinetic crusher."
+	icon_state = "watcher_wing"
+	denied_type = /obj/item/crusher_trophy/watcher_wing
+	bonus_value = 0.7
+
+/obj/item/crusher_trophy/watcher_wing/effect_desc()
+	return "mark detonation to prevent certain creatures from using certain attacks for <b>[bonus_value*0.1]</b> second\s"
+
+/obj/item/crusher_trophy/watcher_wing/on_mark_detonation(mob/living/target, mob/living/user)
+	if(ishostile(target))
+		var/mob/living/simple_animal/hostile/H = target
+		if(H.ranged) //briefly delay ranged attacks

```

Comment:
Does this mean that with a significant enough bonus value, combined with this trophy you can almost completely nullify Hierophant/Colossus?